Dockerfile

```diff
@@ -72,18 +39,53 @@ COPY config/ config/
 RUN chmod u-w docroot/sites/default/settings.php \
   && chmod u-w docroot/sites/default/services.yml
 
+###########################################################################################
+# Create test image
+###########################################################################################
+
+FROM base AS test
+
+# Remove the memory limit for the CLI only.
+RUN echo 'memory_limit = -1' > /usr/local/etc/php/php-cli.ini
+
+# Install dependencies (with dev)
+RUN composer install \
+  --no-ansi \
+  --dev \
+  --no-interaction \
+  --prefer-dist
+
 # Change ownership of files
 RUN chown -R www-data:www-data ./
 RUN chown -R www-data:www-data /var/www
 
 USER www-data
 
-FROM build as local
+FROM test as local
 USER root
 RUN pecl install xdebug-2.9.8 \
   && docker-php-ext-enable xdebug
 USER www-data
```

Comment:
This is nifty. Would we ever want to use `debug` within a test? If so we could add this to the `test` image above.

Reply:
No we wouldn't want to install xdebug on the test build (if that's what you mean), as it really slows things down.  You can only really use it on local environments.